test/test_browser.py

```diff
@@ -4497,21 +4508,9 @@ def test_async_compile(self):
 '''
     shell_with_script('shell.html', 'shell.html', script)
     common_args = ['--shell-file', 'shell.html']
-    for opts, returncode in [
-      ([], 1),
-      (['-O1'], 1),
-      (['-O2'], 1),
-      (['-O3'], 1),
-      # force it on
-      (['-sWASM_ASYNC_COMPILATION'], 1),
-      # force it off. note that we use -O3 here to make the binary small enough
-      # for chrome to allow compiling it synchronously
```

Comment:
Ah, I guess the increase in this limit last year allows this to work even without `-O3` now, nice...

Reply:
Yup!